File: backend/host/Models/Owner.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity;
using System.Text.Json.Serialization;
using host.DataBaseAccess;

namespace host.Models;

public class Owner : TgAccount, ITableDataBase, ITakeRelatedData
{
    [JsonPropertyName("restaurant_ids")] [NotMapped] public List<int>? RestaurantIds { get; set; }

    public Owner(){}
    public Owner(int? id, string username) : base(id, username){}
    public Owner(int? id, string username, List<int>? restaurantIds) : base(id, username)
    {
        RestaurantIds = restaurantIds;
    }

    public async Task TakeRelatedData(ApplicationDbContext context)
    {
        RestaurantIds = await context.Restaurants.Where(r => r.OwnerId == Id).Select(r => r.Id.Value).ToListAsync();
    }
}

File: frontend/Business/tg_business_bot/Models/Owner.py
from typing import List, Optional
from tg_business_bot.Models.TgAccount import TgAccount


class Owner(TgAccount):
    restaurant_ids: Optional[List[int]]

    def __init__(self,
                 id: int,
                 username: str,
                 restaurant_ids: Optional[List[int]] = None,
                 **kwargs):
        super().__init__(id=id,
                         username=username,
                         restaurant_ids=restaurant_ids,
                         **kwargs)
